doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -18,6 +18,25 @@ For detailed information about supported functionalities, see the official `Fast
    The implementation passes tests of `Fast Pair Validator app`_ (beta version).
    The procedure triggered in Android settings is successful (tested with Android 11).
 
+Integration prerequisites
+*************************
+
+Before you start the |NCS| integration with Fast Pair, make sure that the following prerequisites are fulfilled:
+
+* :ref:`Install the nRF Connect SDK <installation>`.
```

Comment:
I would consider adding one more step:
```
* Install requirements of :ref:`bt_fast_pair_provision_script`. 
```
The script is automatically called during build, so it would be good to ensure that user installs all of the dependencies before build

Reply:
updated as : _* Install the requirements mentioned in the :ref:`bt_fast_pair_provision_script` to generate the hexadecimal file required for the :ref:`Google Fast Pair integration in the nRF Connect SDK <ug_bt_fast_pair>`._